spec/system/purchase_system_spec.rb

```diff
@@ -105,16 +109,15 @@
           fill_in "vendor_business_name", with: "businesstest"
           fill_in "vendor_contact_name", with: "test"
           fill_in "vendor_email", with: "123@mail.ru"
-          sleep(0.3)
-          click_on "vendor-submit"
+          find("#vendor-submit").click
```

Comment:
I think this should work? There is one more of these sleep replacements as well.